Library/Formula/swagger-codegen.rb

```diff
@@ -0,0 +1,23 @@
+class SwaggerCodegen < Formula
+  desc "Generation of client and server from Swagger definition"
+  homepage "http://swagger.io/swagger-codegen/"
+  url "https://github.com/swagger-api/swagger-codegen/archive/v2.1.4.tar.gz"
+  sha256 "b4965158fb8abbebdfcaab2fdc0c5556001c587a526279d44ea9878f7ef9ef4c"
+
+  head do
+    url "https://github.com/swagger-api/swagger-codegen.git"
+  end
```

Comment:
Oh yes. Copy'n'paste mistake from another more complex formula. Fix coming up in 2 secs.